terraform/modules/helm/flink/variables.tf

```diff
@@ -156,4 +158,29 @@ variable "flink_sa_annotations" {
   type        = string
   description = "Service account annotations for flink service account."
   default     = "serviceAccountName: default"
+}
+variable "flink_release_map" {
```

Comment:
@ManojCKrishna There needs to be a way to toggle individual jobs or deploy only the merged pipeline and master-data-processor. We don't want both the merged-pipeline and the individual jobs to be deployed separately.